src/Devtech/Common/CliHelper.php

```diff
@@ -9,32 +9,57 @@
 namespace Devtech\Common;
 
 class CliHelper {
+
 	/**
-	 * Parses the arguments passed to CLI and returns the vendorID
-	 *
-	 * @param array $args
-	 * @return int
-	 * @throws \Exception
+	 * @var IValidator
 	 */
+	private $validator;
 
-	public static function getVendorId($args) {
-		if (count($args) > 1 && is_numeric($args[1]) && $args[1] > 0) {
-			return $args[1];
-		} else {
-			throw new \Exception("Argument missing, please provide vendor id as script's first argument.");
+	/**
+	 * CliHelper constructor.
+	 * @param IValidator $validator
+	 * @param $count
+	 */
+	public function __construct(IValidator $validator,$count)
+	{
+		$this->validateNumberOfArgs($count);
+		$this->validator = $validator;
+	}
+
+	/**
+	 * Checks number of arguments passed to CLI
+	 *
+	 * @param integer $count
+	 * @throws Exception
+	 */
+	private function validateNumberOfArgs($count)
+	{
+		if ($count !== 4) {
```

Comment:
I just wanted somehow to validate the number of arguments we are passing to the CLI. Maybe to have in config file how many arguments is required and if users enters less then required to throw an exception. I know that I hard coded it here. Btw I will remove this method.